documentation/Makefile

```diff
@@ -7,25 +7,168 @@ EXPAND_ME += EPICS_REVISION
-GH_FILES = $(ME)/ $(ME)/.nojekyll $(ME)/*.* $(ME)/*/*.*
+database_HEADERS += chfPlugin
+database_HEADERS += dbChannel
+database_HEADERS += dbCommon
+database_HEADERS += dbDefs
+database_HEADERS += dbExtractArray
+database_HEADERS += dbLink
+database_HEADERS += dbServer
+database_HEADERS += dbState
+database_HEADERS += dbStaticLib
+database_HEADERS += dbUnitTest
+database_HEADERS += devLib
+database_HEADERS += devSup
+database_HEADERS += drvSup
+database_HEADERS += initHooks
+
+record_HEADERS += aaiRecord
+record_HEADERS += aaoRecord
+record_HEADERS += aiRecord
+record_HEADERS += aoRecord
+record_HEADERS += aSubRecord
+record_HEADERS += biRecord
+record_HEADERS += boRecord
+record_HEADERS += calcRecord
+record_HEADERS += calcoutRecord
+record_HEADERS += compressRecord
+record_HEADERS += dfanoutRecord
+record_HEADERS += eventRecord
+record_HEADERS += fanoutRecord
+record_HEADERS += histogramRecord
+record_HEADERS += int64inRecord
+record_HEADERS += int64outRecord
+record_HEADERS += longinRecord
+record_HEADERS += longoutRecord
+record_HEADERS += lsiRecord
+record_HEADERS += lsoRecord
+record_HEADERS += mbbiRecord
+record_HEADERS += mbbiDirectRecord
+record_HEADERS += mbboRecord
+record_HEADERS += mbboDirectRecord
+record_HEADERS += permissiveRecord
+record_HEADERS += printfRecord
+record_HEADERS += selRecord
+record_HEADERS += seqRecord
+record_HEADERS += stateRecord
+record_HEADERS += stringinRecord
+record_HEADERS += stringoutRecord
+record_HEADERS += subRecord
+record_HEADERS += subArrayRecord
+record_HEADERS += waveformRecord
+
+menu_HEADERS += menuAlarmSevr
+menu_HEADERS += menuAlarmStat
+menu_HEADERS += menuConvert
+menu_HEADERS += menuFtype
+menu_HEADERS += menuIvoa
+menu_HEADERS += menuOmsl
+menu_HEADERS += menuPini
+menu_HEADERS += menuPost
+menu_HEADERS += menuPriority
+menu_HEADERS += menuScan
+menu_HEADERS += menuSimm
+menu_HEADERS += menuYesNo
+
+HEADER_TYPES = libcom database record menu
+
+HEADER_MD_FILES = $(foreach t, $(HEADER_TYPES), \
+    $(addsuffix _h.md, $($t_HEADERS)))
+API_RST_FILES = $(addsuffix -api.rst, $(HEADER_TYPES))
+
+RTD_SRC = $(COMMON_DIR)/rtd-src
+
+DOCS += README.md
+DOCS += RELEASE_NOTES.md
 
 include $(TOP)/configure/RULES
 
-doxygen: Doxyfile ../mainpage.dox $(INSTALL_HTMLS)
+$(HEADER_MD_FILES): %_h.md: ../HEADER_h.md
+	$(EXPAND_TOOL) -t $(INSTALL_LOCATION) -DHEADER=$* $< $@
+
+$(API_RST_FILES): %-api.rst: ../%-API.rst
+	@$(RM) $@
+	@$(ECHO) Creating $@
+	@$(CP) $< $@
+	@$(foreach h, $($*_HEADERS), \
+	    echo "   $h_h.rst" >> $@;)
+
+ifndef T_A
+doxygen rtd sphinx: inc
+	$(MAKE) -C O.$(EPICS_HOST_ARCH) $@
+else
+
+doxygen: Doxyfile
+	@$(MKDIR) $(RTD_SRC)
 	$(DOXYGEN)
-	rsync -av $(TOP)/html/ html/
 
-.PHONY: doxygen
+rtd: doxygen $(API_RST_FILES) $(HEADER_MD_FILES)
+	rsync -av $(INSTALL_DOC)/ $(RTD_SRC)/
+	rsync -av $(HEADER_MD_FILES) $(RTD_SRC)/
+	rsync -av $(API_RST_FILES) $(RTD_SRC)/
+	rsync -av ../index.rst ../../modules/database/src/std/ComponentReference.md ../conf.py $(RTD_SRC)/
```

Comment:
It should not be necessary to list the `ComponentReference.md` file here, and I especially don't want its relative path to appear. The top-level `make inc` step completes before `make rtd` runs here, and when `make inc` reached `modules/database/src/std` the file will have been installed into the `$(INSTALL_DOC)` directory, so the first `rsync` above should have already copied it into `$(RTD_SRC)`.

Reply:
Done.